lib/app_view.dart

```diff
@@ -47,7 +47,12 @@ class _AppViewState extends State<AppView> with SingleTickerProviderStateMixin {
       body: Padding(
         padding: EdgeInsets.only(bottom: bottomPadding),
         child: LayoutBuilder(builder: (context, constraints) {
-          return desktopView();
+          // return desktopView();
+          return Overlay(
```

Comment:
What does this change do?

Reply:
This was a change I made before the `MaterialApp.home` change because one of my widgets required an Overlay but I believe the `MaterialApp.home` thing takes care of that. I will revert this change.